Dockerfile-dev

```diff
@@ -8,17 +8,26 @@ RUN apt-get update \
     && apt-get autoremove \
     && apt-get clean
 
+ARG backend
+
+ENV PYTHONUNBUFFERED 0
+
+ENV BIGCHAINDB_DATABASE_PORT 27017
+ENV BIGCHAINDB_DATABASE_BACKEND $backend
 ENV BIGCHAINDB_SERVER_BIND 0.0.0.0:9984
 ENV BIGCHAINDB_WSSERVER_HOST 0.0.0.0
 ENV BIGCHAINDB_WSSERVER_SCHEME ws
 
 ENV BIGCHAINDB_WSSERVER_ADVERTISED_HOST 0.0.0.0
 ENV BIGCHAINDB_WSSERVER_ADVERTISED_SCHEME ws
 
-ARG backend
+ENV BIGCHAINDB_START_TENDERMINT 0
```

Comment:
As @ttmc mentioned in his review this `BIGCHAINDB_START_TENDERMINT` is not needed anymore